### PR TITLE
[daggy u] Lint, prettier

### PR DESCRIPTION
**Pull request by hellendag**
## Summary & Motivation

Enable lint and prettier in dagster-university, run them both.

## How I Tested These Changes

Lint, prettier. Verify that the app builds and looks correct.

### CODE REVIEW

**Comment by hellendag** [comment]
Current dependencies on/for this PR:
* master
  * **PR #16374** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/16374" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/16374?utm_source=stack-comment).

**Comment by erinkcochran87** [comment]
@hellendag Forgive the dumb ?, but other than the app looking correct, what else do I need to do to test this?

**Comment by hellendag** [comment]
> @hellendag Forgive the dumb ?, but other than the app looking correct, what else do I need to do to test this?

All of the changes should be cosmetic, and only to the code itself. I think if you want to scan for major incorrect changes to the markdown files (e.g. due to linting or prettier) that would be helpful, but ideally the rendered output of the content should be unchanged by this PR.

**Review by erinkcochran87 — COMMENTED** [review]
Found what I think (hope!) are small, fiddly things - otherwise this looks great. I love that I no longer need to manually fix Notion's weird exported Markdown <3

File: docs/dagster-university/pages/dagster-essentials/lesson-1/why-is-asset-centric-orchestration-good-for-data-engineering.md [hunk 1]
```diff
@@ -103,20 +108,22 @@ With a task-centric approach, you may not know data is outdated until the proces
...
-	2. Re-execute the asset
+
+- Everything was going smoothly, but then something went wrong: your cookies got rubbery because you overmixed them.
+
+  In the real world, think of this as debugging an issue, such as a timeout or data discrepancy.
+
+- 1.  Identify which step in the workflow failed
+  2.  Re-execute the entire pipeline
+- 1. Identify the problematic asset (cookie dough)
+  2. Re-execute the asset
```
[the remark this answers is not in the document]
Comment (hellendag) [reply docs/dagster-university/pages/dagster-essentials/lesson-1/why-is-asset-centric-orchestration-good-for-data-engineering.md:125]: Hmm, I think it ends up formatting correctly when rendered, right? The `*` markers were replaced with `-` everywhere, I think, including these.

File: docs/dagster-university/pages/dagster-essentials/lesson-5/whats-the-definitions-object.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ defs = Definitions(
 
 ---
 
-## Anatomy of the __init__.py file
+## Anatomy of the (`__init__.py`) file
```
[the remark this answers is not in the document]
Comment (hellendag) [reply docs/dagster-university/pages/dagster-essentials/lesson-5/whats-the-definitions-object.md:54]: Oh sorry, this was meant to preserve the monospace style on the filename. I'll just escape the underscores.

File: docs/dagster-university/pages/dagster-essentials/lesson-8/coding-practice-partition-taxi-trips.md [hunk 3]
```diff
@@ -11,17 +11,16 @@ To practice what you’ve learned, partition the `taxi_trips` asset by month usi
 - Because a month’s parquet file may contain historical data from outside the month, it is recommended that you partition by the month of the parquet file, not the month of the trip
 - With every partition, insert the new data into the `taxi_trips` table
 - For convenience, add a `partition_date` column to represent which partition the record was inserted from. You’ll need to drop the existing `taxi_trips` because of the new `partition_date` column. In a Python REPL or scratch script, run the following:
-    
-    ```yaml
-    import duckdb
-    conn = duckdb.connect(database="data/staging/data.duckdb")
-    conn.execute("drop table trips;")
-    ```
+  ```yaml
```
[the remark this answers is not in the document]
Comment (hellendag) [reply docs/dagster-university/pages/dagster-essentials/lesson-8/coding-practice-partition-taxi-trips.md:15]: I added some whitespace, does that help?

Screenshot 2023-09-11 at 3 58 40 PM

File: docs/dagster-university/pages/dagster-essentials/lesson-1/why-is-asset-centric-orchestration-good-for-data-engineering.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (erinkcochran87) [reply docs/dagster-university/pages/dagster-essentials/lesson-1/why-is-asset-centric-orchestration-good-for-data-engineering.md:126]: It looks good now - either my eyes got crossed here or your table fix addressed this. Either way, it looks good now!

File: docs/dagster-university/pages/dagster-essentials/lesson-2/project-files.md [hunk 4]
```diff
@@ -41,61 +41,81 @@ The columns in the following table are as follows:
...
-    A Python module that will contain your Dagster code. This directory also contains the following:
 
-    - `__init__.py` - This file includes a `Definitions` object that defines that is loaded in your project, such as assets and sensors. This allows Dagster to load the definitions in a module. We’ll discuss this topic, and this file, later in this course.
+- **dagster_university/**
+- Dagster
+- A Python module that will contain your Dagster code. This directory also contains the following:
+
+  `__init__.py` - This file includes a `Definitions` object that defines that is loaded in your project, such as assets and sensors. This allows Dagster to load the definitions in a module. We’ll discuss this topic, and this file, later in this course.
+
+  Several directories, for example: `/assets`. These directories follow our recommended best practices and will be used to contain the definitions - like assets - you create in the following lessons. We’ll discuss the files they contain later, too.
```
[the remark this answers is not in the document]
Comment (erinkcochran87) [reply docs/dagster-university/pages/dagster-essentials/lesson-2/project-files.md]: I'm not sure if this is a Vercel issue or not, but this copy still looks like this for me:

Screenshot 2023-09-11 at 5 17 20 PM

I tried to pull this down locally to check it out, but got yelled at by a bunch of merge conflicts and didn't want to commit anything to your branch.

**Review by erinkcochran87 — COMMENTED** [review]
🏓

File: docs/dagster-university/pages/dagster-essentials/lesson-2/project-files.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (hellendag) [reply docs/dagster-university/pages/dagster-essentials/lesson-2/project-files.md]: Yeah, something's up with the Vercel preview again. Here's what it looks like locally:

image

**Review by erinkcochran87 — APPROVED** [review]

**Pull request merged**